chameleon/templates/dashboard.html

```diff
@@ -70,18 +70,18 @@ <h4 class="panel-title pull-left">Active Projects</h4>
                       </td>
                       <td>
                         {% for a in project.allocations %}
-                          {% if a.resourceId == alloc.resourceId %}
```

Comment:
This is going to break compatibility with the current UI, which works against TAS. We need to either make this UI work for both somehow (maybe shimming the TAS model with attributes like `date_reviewed` which map to their internal properties (`dateReviewed`)? Happy to discuss in more detail.